### PR TITLE
 chore: adding labels, fix: value proxyPort

### PR DESCRIPTION
**Pull request by deyaeddin**
- this is a version bump only, 
- needs to release chart and docker image

### CODE REVIEW

**Comment by aleveille** [comment]
Hi @deyaeddin!

It seemed I've missed your PR, sorry about that. After review, even if the version bump isn't required anymore the other commits ('[fix: value proxyPort](https://github.com/aleveille/octant-dashboard-turnkey/pull/12/commits/2c8e4b8049f7dcf9bc7d05e99e69bb284e58c0ce) ' and '[chore: adding lables to clusterrole.yaml](https://github.com/aleveille/octant-dashboard-turnkey/pull/12/commits/728f62b83a267fbda8092677a359853dcff852a4)') are interesting.

If you'd like, please rebase your PR to include only those two commits and I'll gladly merge.

Thanks,
Alexandre

**Comment by deyaeddin** [comment]
Hey @aleveille 
I have removed the version bump and kept the other commits only!

Cheers,
Deya

File: helm/values.yaml [hunk 1]
```diff
@@ -97,3 +97,4 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+{"mode":"full","isActive":false}
```
Comment (aleveille) [inline helm/values.yaml]: What is this line about?

Comment (deyaeddin) [reply helm/values.yaml]: Sorry my bad ... copy/pasting sucks ;-)

**Review by aleveille — APPROVED** [review]

**Pull request merged**